### PR TITLE
Clean up logging

### PR DESCRIPTION
**Pull request by cmurtaugh**
This PR reduces several `warning` logs to `debug`. 

Closes TLT-4078.

### CODE REVIEW

File: dj_secure_settings/loader.py [hunk 1]
```diff
@@ -63,9 +63,9 @@ def load_secure_settings(project_name=None, environment=None):
         yaml_params = yaml.load(open(yaml_file), Loader=yaml.Loader)
         _load_params_from_yaml(config, yaml_params, env, 'defaults')
         _load_params_from_yaml(config, yaml_params, env, project_name)
-    except Exception as e:
+    except FileNotFoundError as e:
         # couldn't load params from a local file
-        logging.warning("Couldn't load params from local file: {}".format(str(e)))
+        logging.debug("Couldn't load params from local file: no local file exists")
```
Comment (elliottyates) [inline dj_secure_settings/loader.py]: I recall that every time I get this wrong I have to go look in the code to figure out what the file should be, or troubleshoot why it's not finding the file I think it should -- could we update the message to include `yaml_file`? That might speed up troubleshooting when we have erroneous assumptions about those values. Also, maybe this message can say "File `yaml_file` not found" or something similar, to distinguish it better from the message on line 85?

File: dj_secure_settings/loader.py [hunk 2]
```diff
@@ -82,7 +82,7 @@ def _load_params_from_yaml(config, yaml_params, env, namespace):
             config[k] = yaml_params[env][namespace][k]
     except KeyError:
         # couldn't load the parameters
-        logging.warning("Was not able to load params from a local YAML file.")
+        logging.debug("Was not able to load params from a local YAML file.")
```
Comment (elliottyates) [inline dj_secure_settings/loader.py]: Similar to the comment for line 68,  could we update the message to include `env` and `namespace`? That might speed up troubleshooting when we have erroneous assumptions about those values. Also, maybe this message can say "Unable to load keys for env `env` in namespace `namespace`" or something similar, to distinguish it better from the message on line 68 and help the developer fix the issue more quickly.

**Review by elliottyates — APPROVED** [review]
Cool! A couple of requests that would help me when I run into configuration issues in the future. Thanks!

File: dj_secure_settings/loader.py (hunk 1 above)
Comment (cmurtaugh) [reply dj_secure_settings/loader.py]: Sure -- I'll add that.

**Pull request merged**